shared/aggregation/attestations/maxcover.go

```diff
@@ -37,13 +37,7 @@ func MaxCoverAttestationAggregation(atts []*ethpb.Attestation) ([]*ethpb.Attesta
 		}
 
 		// Find maximum non-overlapping coverage.
-		maxCover, err := NewMaxCover(unaggregated)
-		if err != nil {
-			if errors.Is(err, aggregation.ErrBitsDifferentLen) {
-				return atts, nil
-			}
-			return aggregated.merge(unaggregated), err
-		}
```

Comment:
`NewMaxCover()` doesn't return error now, since it is used in a loop over the same set of attestations, and it is more optimal to validate that set of attestations once, before loop starts (which we indeed do, see line 25 above).